mitodl/mitxonline
# 3400 remove enrolled state and modal default

### What are the relevant tickets?




part of https://github.com/mitodl/hq/issues/3404 https://github.com/mitodl/hq/issues/3400

### Description (What does it do?)

- Remove the "Enrolled" box, so it always display Enroll Now button.
- When there are multiple courseruns open for enrollment, the enrollment modal should default to the courserun with the earliest start date, even if it is in the past.

### How can this be tested?

- Create two runs that both open for enrollments. e.g.
<img width="1227" alt="image" src="https://github.com/mitodl/mitxonline/assets/3138890/5216ed70-80d9-4d68-b495-0b42622407b8">

- Create CMS course page and products for these two runs

- Go to course about page and click on Enroll now, it defaults to Jan 16 run 
<img width="1111" alt="image" src="https://github.com/mitodl/mitxonline/assets/3138890/1bde3f52-decb-4b16-ae23-ad3058cd10a2">

-  Enroll and Pay for this run
- Attempt to enroll it again (Either Enroll for free or Enroll and Pay), you should get message indicated that you have already paid for
<img width="1226" alt="image" src="https://github.com/mitodl/mitxonline/assets/3138890/213ba0f9-778c-4e30-86fd-af7c5d7ef777">

In case of refunded, it should be ok since this PR does't change the flow/logic for "Enroll and Pay"
